fpga_builder/utils.tcl

```diff
@@ -477,10 +524,24 @@ proc build_device_from_params {params} {
   set synth_strategy [dict get $params synth_strategy ]
   set impl_strategy [dict get $params impl_strategy ]
   set origin_dir [dict get $params origin_dir]
-  set use_power_opt [dict_get_default $params use_power_opt 1]
-  set use_post_route_phys_opt [dict_get_default $params use_post_route_phys_opt 1]
+  set auto_incremental_checkpoint [dict get $params auto_incremental_checkpoint]
```

Comment:
I'm not sure that all of these need exposed to the builder.

auto_incremental_checkpoint, is_enabled, max_paths, need_refresh, don't seem needed.  The others might but I am not sure.